package.json

```diff
@@ -14,9 +14,7 @@
     "lintTest": "eslint test/. --ext .js",
     "lintTest:fix": "eslint test/. --ext .js --fix",
     "clean": "rimraf ./dist/resources/ && mkdir ./dist/resources",
-    "images-nav-frontend": "cp -R node_modules/nav-frontend-grafikk dist/resources/img",
-    "images-local": "cp -R img dist/resources",
-    "images": "npm-run-all images-nav-frontend images-local",
+    "images": "cp -R img dist/resources",
```

Comment:
Klarer ikke helt og se hva dette gjør men du har vel kontroll? :P

Reply:
Jeg fjerna en avhengighet som egentlig bare kopierte inn to ikoner. Kopierte de ikonene inn i img filen så trenger ikke dobbelt opp.